src/models/Item.js

```diff
@@ -500,6 +502,9 @@ export default class Item {
       'responseOptions.requiredValue': {
         updated: optionUpdate('Required option'),
       },
+      'responseOptions.showTickMarks': {
+        updated: optionUpdate('Required option'),
```

Comment:
will you please update this one to "show tick marks"?

Reply:
other changes are looking really good to me.

Reply:
Resolved.